src/places/places.controller.ts

```diff
@@ -87,7 +89,21 @@ export class PlacesController {
         return await this.demandsService.getDetailedDemandsForPlace(transaction, id);
       });
     } catch (error) {
-      throw new HttpException('Cannot get demands for place', HttpStatus.BAD_REQUEST);
+      throw new HttpException(`Cannot get demands for place ${id}`, HttpStatus.BAD_REQUEST);
+    }
+  }
+
+  @SetMetadata(MetadataKey.ALLOWED_ROLES, [UserRole.ADMIN, UserRole.PLACE_MANAGER])
+  @UseGuards(AuthGuard)
+  @HttpCode(HttpStatus.NO_CONTENT)
+  @Delete(':id/demands')
+  public async deleteDemandsForPlace(@Param('id') id: string): Promise<void> {
+    try {
+      await this.sequelize.transaction(async (transaction): Promise<void> => {
+        await this.demandsService.deleteAllDemandsForPlace(transaction, id);
+      });
+    } catch (error) {
+      throw new HttpException(`Cannot delete demands for place ${id}`, HttpStatus.BAD_REQUEST);
```

Comment:
```suggestion
      throw new HttpException('CANNOT_DELETE_DEMANDS', HttpStatus.BAD_REQUEST);
```

This will be easily mapped to an enum and later on handled by the application :)

Reply:
OK, let's do it this way :)